src/SurveyHandler.ts

```diff
@@ -786,6 +786,7 @@ class SurveyHandler {
             // let endRangeLabel = "Very likely";
             let endRangeLabel = "";
             
+            let customMetric = this.surveyData.customMetrics.sample;
```

Comment:
Sample is the default custom metric created by the program itself. It is not a input taken by the user. Since only one customMetric can be created per user, and further it can only be edited for any questionnaire, the attribute 'sample' of the surveyData.customMetrics remains sample throughout. This is what I have gained, can you please crosscheck and confirm.